src/sentry/relay/config/__init__.py

```diff
@@ -159,9 +159,7 @@ def get_project_config(
 
 
 def get_dynamic_sampling_config(project: Project) -> Optional[Mapping[str, Any]]:
-    feature_multiplexer = DynamicSamplingFeatureMultiplexer(project)
-
-    if feature_multiplexer.is_on_dynamic_sampling:
+    if is_on_dynamic_sampling(project):
```

Comment:
The idea is to use directly: like we do across all sentry code base:
```
 features.has("organizations:dynamic-sampling", project.organization) and options.get(
        "dynamic-sampling:enabled-biases"
    )
```

Reply:
just grep by: `features.has("organizations:`

Reply:
I didn't like this approach as soon as I saw it used throughout the codebase, but if we you think that we should unify, then let's do that

Reply:
python -c "import this" | rg exp
4:Explicit is better than implicit.

🤷

Reply:
I think that a proper design of the features would be to allow for composite ones but that is another story.

Reply:
feature flagger already supports it.

Reply:
I meant a composite that would span different contexts and for example automatically get access to options, that would be quite nice